src/snapshot-editor/src/info.rs

```diff
@@ -32,6 +32,12 @@ pub enum InfoVmStateSubCommand {
         #[arg(short, long)]
         vmstate_path: PathBuf,
     },
+    /// Print readable MicroVM state.
```

Comment:
commit messages: please add the scope (snapshot editor) to make it clear that it is not FC that gets the new option.